docs/source/examples/tags.md

```diff
@@ -0,0 +1,22 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+orphan: true
+---
+
+# Example Tags
```

Comment:
nit: this title may be misread as "tags as example" rather than "tags for example", if you see what I mean here. Perhaps just titling as «Tags» is better.